static/network-status.json

```diff
@@ -1,4 +1,6 @@
 {
     "networkStatusIndicator": 1,
-    "_comment": "Possible statuses: '1' - Operational; '2' - Degraded Performance; '3' - Network Unavailable"
+    "_comment_networkStatusIndicator": "Possible statuses: '1' - Operational; '2' - Degraded Performance; '3' - Network Unavailable",
+    "underMaintenanceIndicator": 0,
```

Comment:
I would rename this to `isBatchExplorerUnderMaintenance` as we would need another one for the wallet in the short term. Also a boolean in this case probably is better than a number indicating a status as it can also have two possibles values.

Reply:
Done